### PR TITLE
interpolate

### PR DESCRIPTION
**Pull request by hussain-jafari**
### Description
- *Category*: feature
- *JIRA issue*: https://jira.ihme.washington.edu/browse/MIC-5201

### Changes and notes
Linearly interpolate RRs using lookup table output. 

### Testing
Spot checked a few RRs using simulant exposure values. Plotted interpolated results over original RR data and checked that the shapes were the same visually. A representative example is shown below:

interpolated_rrs
original_rrs

### CODE REVIEW

File: src/vivarium_public_health/risks/effect.py [hunk 1]
```diff
@@ -468,10 +469,15 @@ def get_target_modifier(
...
-            # use exposures and rr points to calculate m and b in y=mx+b
-            # ie m = (y2-y1)/(x2-x1) and b = y1 - mx1
-            # plug exposures into mx+b to get our RRs
+            exposure = self.population_view.get(index)[f"{self.risk.name}_exposure"]
+            x1, x2 = (
+                rr_intervals["left_exposure"].values,
+                rr_intervals["right_exposure"].values,
+            )
+            y1, y2 = rr_intervals["left_rr"].values, rr_intervals["right_rr"].values
+            m = (y2 - y1) / (x2 - x1)
```
Comment (stevebachmeier) [inline src/vivarium_public_health/risks/effect.py:478]: Is it possible to have x2 = x1 and get a DivideByZero?

**Review by stevebachmeier — APPROVED** [review]

**Comment by patricktnast** [comment]
>  Plotted interpolated results over original RR data and checked that the shapes were the same visually.

It would be awesome to post those plots here!

**Review by patricktnast — APPROVED** [review]

File: src/vivarium_public_health/risks/effect.py (hunk 1 above)
Comment (hussain-jafari) [reply src/vivarium_public_health/risks/effect.py:478]: Good question. x1 is the largest exposure for which an RR is defined that is less than or equal to a simulant's exposure and x2 is the smallest exposure for which an RR is defined that is greater than simulant's exposure, so they should never be equal.

**Comment by hussain-jafari** [comment]
Good call @patricktnast I've attached some pictures in the description

**Pull request merged**